travis/Dockerfile-qt5-travis

```diff
@@ -0,0 +1,15 @@
+FROM openms/contrib:qt5-v2.0
+
+WORKDIR /
+ADD . /OpenMS
+RUN cd /OpenMS 
```

Comment:
so how exactly does the OpenMS source code from the pull-request enter the container? does that happen here?